### ISSUE TITLE
Changing font of quick reply buttons

### ISSUE THREAD

**Issue by admehta01** [FaridSafi/react-native-gifted-chat#2171]
#### Issue Description

Unable to change font-family/size of quick reply buttons

#### Steps to Reproduce / Code Snippets
```
  const renderQuickReplies = (props: any) => (
    <QuickReplies
      {...props}
      color="purple"
      quickReplyStyle={{
        fontSize: 18,
        borderWidth: 2,
        borderRadius: 10,
        marginRight: 5,
        marginVertical: 5,
      }}
    />
  );
```

#### Expected Results

Text should be larger in quick reply bubbles.

#### Additional Information

* Nodejs version: 16.14.2
* React version: 17.0.1
* React Native version: 0.64.3
* react-native-gifted-chat version: 0.16.3
* Platform(s) (iOS, Android, or both?): Android
* TypeScript version: 4.3.5

**Comment by Johan-dutoit** [comment]
`quickReplyStyle` is of type `ViewStyle` and only affects the 'container".
Currently there is no way to change the style of the buttons directly. 

One way would be to override the entire quick replies component.

**Comment by Johan-dutoit** [comment]
Feature added, will be available in `v1.0.0-beta-3`

**Comment by admehta01** [comment]
Thank you!
